### PR TITLE
Adding bibmon

### PR DESCRIPTION
**Pull request by camaramm**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by camaramm** [comment]
@conda-forge/staged-recipes

**Comment by camaramm** [comment]
@conda-forge/help-python

File: recipes/bibmon/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (ocefpaf) [inline recipes/bibmon/LICENSE.md:1]: This file is in the tarball. No need to add it here.

File: recipes/bibmon/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bibmon-{{ version }}.tar.gz
+  sha256: d2cd59ca1b1b32224e03a3a610f0ced3d15bb3f459835d652c147ecf12439529
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/bibmon/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Comment by ocefpaf** [comment]
At first I thought that the metadata was wrong b/c there is no requirements in it. However, inspecting the code, I found this in the `setup.py`:

```python
requirements = os.path.join(
    os.path.dirname(os.path.realpath(__file__)),
    'requirements.txt'
)

if os.path.isfile(requirements):
    with open(requirements) as f:
        install_requires = f.read().splitlines()
else:
    install_requires = []
```

If you can please open an issue upstream to either ship the `requirement.txt` file in the tarball or migrate to a modern Python metadata, where you declare the dependencies.

**Comment by camaramm** [comment]
Sure, I will improve as suggested. Thanks

File: recipes/bibmon/LICENSE.md (hunk 1 above)
Comment (camaramm) [reply recipes/bibmon/LICENSE.md:1]: Ok

Comment (ocefpaf) [reply recipes/bibmon/LICENSE.md:1]: Please mark as resolved when it is resolved are we will lose track of things.

File: recipes/bibmon/meta.yaml (hunk 2 above)
Comment (camaramm) [reply recipes/bibmon/meta.yaml:14]: No need to change here, as stated in the comment https://github.com/conda-forge/staged-recipes/pull/27400#issuecomment-2314430863

Comment (ocefpaf) [reply recipes/bibmon/meta.yaml:14]: This is a required change and is unrelated to that comment. This is the script line we use in conda-forge to ensure we won't get accidental wheels into the mix.

Comment (camaramm) [reply recipes/bibmon/meta.yaml]: I'm sorry for the misunderstanding. Done.

**Pull request merged**